app/src/main/java/com/kickstarter/viewmodels/ProjectSearchResultHolderViewModel.java

```diff
@@ -0,0 +1,111 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.util.Pair;
+
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.rx.transformers.Transformers;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.ui.viewholders.ProjectSearchResultViewHolder;
+
+import rx.Observable;
+import rx.subjects.BehaviorSubject;
+import rx.subjects.PublishSubject;
+
+public interface ProjectSearchResultHolderViewModel {
+
+  final class Data {
+    final Project project;
+    final boolean isFeatured;
+
+    public Data(final @NonNull Project project, final boolean isFeatured) {
+      this.project = project;
+      this.isFeatured = isFeatured;
+    }
+  }
+
+  interface Inputs {
+    /** Call to configure the view model with a message. */
+    void configureWith(Data data);
+
+    // FIXME: add documentation
+    void onClick();
```

Comment:
i'd say adding documentation, as well as renaming this to a more descriptive input name would be 💯 
i.e. `somethingClicked();`